datadog_checks_dependency_provider/datadog_checks/datadog_checks_dependency_provider/data/conf.yaml.example

```
@@ -1,4 +1,4 @@
-init_config: {}
+init_config:
```

```suggestion
init_config: {}
```
Let's not modify unnecessary integrations